pkg/bundle/internal/ssa_client/patch.go

```diff
@@ -17,10 +17,101 @@ limitations under the License.
 package ssa_client
 
 import (
+	"encoding/json"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/types"
+	coreapplyconfig "k8s.io/client-go/applyconfigurations/core/v1"
+	v1 "k8s.io/client-go/applyconfigurations/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	trustapi "github.com/cert-manager/trust-manager/pkg/apis/trust/v1alpha1"
+)
+
+const (
+	FieldManager = client.FieldOwner("trust-manager")
```

Comment:
This constant seems a bit "misplaced", but I didn't want to create a new Go-file for just this...